{"uuid": "a00bf218-2f13-451b-a8d3-2f28801cdfb9", "children": ["9842ca2d-0ce1-4f7f-9677-e70fc0207ee6"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='localhost', port=4444): Max retries exceeded with url: /wd/hub/session (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DF7E4EA600>: Failed to establish a new connection: [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение'))\n", "trace": "  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 877, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\tests\\conftest.py\", line 17, in driver\n    driver = webdriver.Remote(command_executor=grid_url, options=options)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 208, in __init__\n    self.start_session(capabilities)\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 292, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 345, in execute\n    response = self.command_executor.execute(driver_command, params)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 302, in execute\n    return self._request(command_info[0], url, body=data)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 322, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\urllib3\\_request_methods.py\", line 144, in request\n    return self.request_encode_body(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\urllib3\\_request_methods.py\", line 279, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\urllib3\\poolmanager.py\", line 444, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 877, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 877, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 877, in urlopen\n    return self.urlopen(\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\urllib3\\connectionpool.py\", line 847, in urlopen\n    retries = retries.increment(\n              ^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\V\\PycharmProjects\\SimbirSoft\\.venv\\Lib\\site-packages\\urllib3\\util\\retry.py\", line 515, in increment\n    raise MaxRetryError(_pool, url, reason) from reason  # type: ignore[arg-type]\n    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1715673391977, "stop": 1715673408095}], "start": 1715673391977, "stop": 1715673408354}